packages/client/src/index.ts

```diff
@@ -1 +1,11 @@
-export default {};
+import {Property, Signature} from "../../message/src/index";
+
+// TODO: @ts-ignore should be removed, when below function is implemented.
+// @ts-ignore
+export async function register_info(url: string, properties: Property[]): Promise<Signature> {
```

Comment:
Because only the skeleton was created, a type error of return value occurs. Later, when implementation is complete, we should remove the `@ts-ignore`.